app/models/manifest.rb

```diff
@@ -5,6 +5,12 @@
 class Manifest < ActiveRecord::Base
   validates :tracking_number, presence: true
 
+  before_validation :set_tracking_number, on: :create
+
+  def set_tracking_number
```

Comment:
since we already validate against tracking number in the json, what about *not* validating the field itself so we don't have to deal with a callback?